deploy/kubernetes/snapshot/validation-webhook/README.md

```diff
@@ -0,0 +1,185 @@
+# Snapshot validation webhook
+For a better understanding of webhooks and validation webhook in kubernetes, please refer  [this](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).
+
+In simple terms, the snapshot validation webhook when deployed, validates the input provided for creating/updating snapshot as described below:
+1. Without the webhook, although the snapshot won't get created, no error is returned.
+2. Kubernetes will return error when you try to update the snapshot with a different pvc name. Without the webhook, no error is returned, but snapshot config will be updated to new pvc given, but it will still be a snapshot of the previous pvc.
+
+To understand more about snapshot validation web hook and it's benefits, refer [snapshot validation webhook]( https://github.com/kubernetes-csi/external-snapshotter/tree/v6.0.1#validating-webhook).
+
+## Deploying snapshot validation webhook
+**Note:** Out of the two ways defined below, deploying wihout cert manager works on Kubernetes clusters, but is not verified on openshift clusters. Deploying with cert manager works on both kubernetes and openshift clusters.
+
+### Deploying webhook without cert manager
+**Note:** This is not considered to be a production ready method to deploy the certificates. This is only one of many ways to deploy the certificates, it is your responsibility to ensure the security of your cluster. TLS certificates and private keys should be handled with care and you may not want to keep them in plain Kubernetes secrets.
+
+The steps to deploy this webhook is already provided [here](https://github.com/kubernetes-csi/external-snapshotter/blob/v6.0.1/deploy/kubernetes/webhook-example/README.md#how-to-deploy-the-webhook), but the same might not work on your cluster, hence follow these steps (these are the same steps provided in this [READme](https://github.com/kubernetes-csi/external-snapshotter/blob/v6.0.1/deploy/kubernetes/webhook-example/README.md#how-to-deploy-the-webhook) but with some modifications and more details).
```

Comment:
can `READme` be of same case letters

Reply:
Addressed https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/pull/97/files#diff-8f2ca0dca69185ccc1d65318877a2beab636847470d83a4cbdb13f41d1d0859bR11 - Added a note here

Reply:
And updated READ me too